resources/language/resource.language.en_gb/strings.po

```diff
@@ -145,6 +145,10 @@ msgctxt "#30047"
 msgid "PRESEASON WEEK {0}"
 msgstr ""
 
+msgctxt "#30047"
+msgid "PRESEASON WEEK {0}"
+msgstr ""
+
```

Comment:
This is a duplicate now, I assume you copied it by mistake?

Reply:
Yes, did not notice that. Should not be duplicated.